src/__test__/render.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import Rockets from '../components/Rockets';
+import Profile from '../components/profile';
+import Missions from '../components/Missions';
+import store from '../redux/configureStore';
+
+describe('Space App components rendering test', () => {
+  it('match snapshot for Rockets', () => {
+    const tree = TestRenderer.create(
+      <Provider store={store}>
+        <Rockets />
+      </Provider>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('match snapshot for the Missions', () => {
+    const tree = TestRenderer.create(
+      <Provider store={store}>
+        <Missions />
+      </Provider>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('match snapshot for the Profile', () => {
+    const tree = TestRenderer.create(
+      <Provider store={store}>
+        <Profile />
+      </Provider>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Kindly add tests to test all features and components per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=NOTE%3A%20Make%20sure%20to%20add%20test%20for%20all%20your%20components%20and%20functions.) not only test snapshots and make sure that all tests run successfully.

Reply:
Done :heavy_check_mark: